src/atoms/Article.js

```diff
@@ -1,8 +1,13 @@
-import styled from 'react-emotion';
+import styled, { css } from 'react-emotion';
 import { getVariable } from '../utils';
 
-const Article = styled.article`
+const articleCss = props => css`
```

Comment:
I don't understand why you need to export `articleCss`, this looks like an antipattern.

Reply:
Still unnecessary changes in this file. Consider rebasing onto master?

Reply:
These changes are needed to make the floating effect on plugs in articles work.